rest/api.go

```diff
@@ -346,6 +348,13 @@ func (h *handler) handlePprofProfile() error {
 }
 
 func (h *handler) handlePprofBlock() error {
+	sec, err := strconv.ParseInt(h.rq.FormValue("seconds"), 10, 64)
+	if sec <= 0 || err != nil {
+		sec = 30
+	}
+	runtime.SetBlockProfileRate(1)
```

Comment:
Might need to add some blocking to only allow a single profile to be running at a time, otherwise they'll stomp over each other's BlockProfileRates/MutexProfileFractions.